### DISCUSSION TITLE
Sonof S26R2ZB Power Behavior Settings

### DISCUSSION THREAD

**Discussion by Merril-r** [Koenkk/zigbee2mqtt#16792] [Devices]
Anybody experienced the same issue on this Sonof Zigbee plug.

When going to the exposes tab and setting the power on behavior I get the error below. Is this the plug or an issue on the side of Z2m

Thanks in advance.

![878A2BBE-9742-4CB1-AE91-DC921B7747E1](https://user-images.githubusercontent.com/99876849/220865468-302e42dd-b86e-4a31-800c-ed81bc0d2e9e.jpeg)

**Comment by Koenkk** [comment]
Seems the device does not support this, I've disabled it.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by Merril-r** [reply]
@Koenkk Thanks for the reply. That is really a pity that the device does not support the feature.
